dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1339,17 +1340,41 @@ class WebCompileTest {
     });
   }
 
-  /// Obtains the size and gzipped size of a file given by [fileName].
-  static Future<Map<String, int>> getSize(String fileName, {required String metric}) async {
+  /// Obtains the size and gzipped size of both [dartBundleFile] and [buildDir].
+  static Future<Map<String, int>> getSize(
+    String buildDir,
+    String dartBundleFile, {
+    required String metric,
+  }) async {
     final Map<String, int> sizeMetrics = <String, int>{};
 
-    final ProcessResult result = await Process.run('du', <String>['-k', fileName]);
+    // get size of dart.js bundle
+    ProcessResult result = await Process.run('du', <String>['-k', dartBundleFile]);
     sizeMetrics['${metric}_dart2js_size'] = _parseDu(result.stdout as String);
```

Comment:
I think the incoming array of files will need to be a map of String fileName -> String metricName, so we don't lose continuity with the metrics this has been gathering until now :/

Or this could generate the name of the metric from the filename and automate it forever :P

Reply:
you must also be a rubyist :P